model_analyzer/result/result_manager.py

```diff
@@ -160,9 +170,50 @@ def create_tables(self, gpu_specific_metrics, non_gpu_specific_metrics):
             The metrics that do not have a GPU id associated with them
         """
 
+        gpu_specific_metrics_from_measurements = {}
```

Comment:
Would like to see a qa test for this. Also please add a couple of lines of comments explaining why it is necessary for us to get the metrics from the checkpoint here.

Reply:
Explanation is added.

Reply:
Could the `L0_export_metrics` test this case? It runs the `analyze` command on a pre-profiled checkpoint file, which includes this logic during the run. If the output metrics are correct, then this logic should have captured the metric columns correctly.